extensions/2.0/Khronos/KHR_materials_ior/README.md

```diff
@@ -0,0 +1,87 @@
+# KHR\_materials\_ior
+
+## Khronos 3D Formats Working Group
```

Comment:
Please change this line to `## Contributors`.  If possible, add a contributor list similar to [the one in Clearcoat](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_materials_clearcoat#contributors).

Please also copy the [Appendix from Clearcoat](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_materials_clearcoat#appendix-full-khronos-copyright-statement).